permissions/plugin-ibm-cloud-devops.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "ibm-cloud-devops"
+paths:
+- "com/ibm/devops/ibm-cloud-devops"
+developers:
+- "xunrongl"
+- "imvijay2007"
```

Comment:
@daniel-beck thank you for catching that.  I just confirmed with both of them that they've signed into Artifactory.